src/connectd/main.mo

```diff
@@ -1,4 +1,4 @@
-import Digraph "./digraph";
+import Digraph "./Digraph";
```

Comment:
```suggestion
import Digraph "./digraph";
```
Refer to this issue: https://github.com/dfinity-lab/motoko/issues/1435
tl;dr This works on Mac, but will break Linux. I don't know a good way to fix this.